src/bundle/Resources/views/themes/admin/ui/form_fields.html.twig

```diff
@@ -1,10 +1,18 @@
 {% extends 'bootstrap_4_layout.html.twig' %}
 
 {% block form_label -%}
-    {%- set label_attr = label_attr|merge({class: (label_attr.class|default('') ~ ' ez-label')|trim}) -%}
-    {{ parent() }}
+    {%- set label_attr = label_attr|merge({class: (label_attr.class|default('') ~ ' ibexa-label')|trim}) -%}
+    {%- set is_not_only_errors = label is not same as(false) -%}
+    {% if is_not_only_errors -%}
```

Comment:
Not sure if making it `{% if not is_only_errors -%}`, personally I don't like this kind of negations in variables